flexmeasures/cli/data_add.py

```diff
@@ -1314,6 +1316,13 @@ def add_schedule_process(
     type=click.File("r"),
     help="Path to the JSON or YAML file with the configuration of the reporter.",
 )
+@click.option(
+    "--source-id",
+    "source",
+    required=False,
+    type=DataSourceIdField(),
+    help="DataSource ID of the `Reporter`.",
+)
```

Comment:
Good idea!

Firstly, I tried to register the data sources in the creation of the app but  this would fail in case of having a DB setup beforehand (tables created).

Threfore, I though we can control more where/when to add them via a CLI command. The command adds a `DataSource` for each of the reporters registered in data_generators. I'm holding on the creation of a CLI changelog entry until we decide wheter this change is going in.